src/int-util.h

```diff
@@ -119,6 +119,11 @@ static inline uint32_t div128_32(uint64_t dividend_hi, uint64_t dividend_lo, uin
 static inline uint32_t ident32(uint32_t x) { return x; }
 static inline uint64_t ident64(uint64_t x) { return x; }
 
+#undef swap32
+#define swap32 swap32
+#undef swap64
+#define swap64 swap64
```

Comment:
You missed `john_` there.

Reply:
sorry for this messup, built a different tree than I pushed, should be fixed now.